src/Apps/__stories__/OrderApp.story.tsx

```diff
@@ -11,33 +12,9 @@ const mock = {
   }),
   Order: (_, { id, ...others }) => {
     return {
+      ...IncompleteOrder,
```

Comment:
#minor `Incomplete` is a little ambiguous-- does this refer to an order with no shipping? No payment details? Or just one that hasn't been submitted yet?